ios/scripts/validate-app-store-release.sh

```
@@ -46,6 +46,12 @@ EOF
 die() { printf 'validate-app-store-release: %s\n' "$*" >&2; exit 1; }
 note() { printf 'validate-app-store-release: %s\n' "$*" >&2; }
 
+read_xcconfig_setting() {
+  local key="$1"
+  local file="$2"
+  sed -nE "s/^[[:space:]]*$key[[:space:]]*=[[:space:]]*([^[:space:]]+).*/\\1/p" "$file" 2>/dev/null | tail -n 1
```

Shellcheck (0.11.0)

[error] Use braces when expanding arrays, e.g. ${array[idx]} (or ${var}[.. to quiet).

(SC1087)